synapse/handlers/devicemessage.py

```diff
@@ -47,7 +47,9 @@ def __init__(self, hs: "HomeServer"):
         self.is_mine = hs.is_mine
 
         # We only need to poke the federation sender explicitly if its on the
-        # same instance.
+        # same instance. Other federation sender instances will get notified by
+        # the `generic_worker` when it sees it in the to-device replication
```

Comment:
sorry, what's the `generic_worker` here?

Reply:
Specifically the federation sender in the `synapse.app.genreic_worker` file. Will expand